modules/virt-disabling-node-image-limits.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assembly:
+//
+// * virt/node_maintenance/virt-node-image-limits.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-disabling-node-image-limits_{context}"]
+= Disabling node image limits
+
+If your pod has more than 50 images, you can disable the image limit by editing the `KubeletConfig` object.
+
+.Prerequisites
+* Create a `kubeletconfig` custom resource (CR)
+
+.Procedure
+
+* In the `kubeletConfig` CR, set the value of of `nodeStatusMaxImages` to `-1`.
```

Comment:
Can we link to any place in the OCP docs here?

Reply:
@fabiand Unfortunately we can't link inside the modules, only the assembly that holds it. Is there anything you'd like to add to the Additional Resources section?